### DISCUSSION TITLE
Finding PTZ compatible cameras

### DISCUSSION THREAD

**Discussion by aneisch** [blakeblackshear/frigate#7997] [Ask A Question]
Does anyone have a _list_ of compatible PTZ cameras or a specific one they recommend?

**Comment by NickM-27** [comment]
I personally have found amcrest/dahua cameras and reolink cameras to be compatible

**Reply by hawkeye217** [reply]
+1 for Amcrest and Dahua PTZs, as @NickM-27 said. Some other users have reported success with Hikvision PTZs, but others have had huge problems. Tapo PTZs have also been problematic.

Also, avoid gray market cameras that tell you to not upgrade the firmware.

**Reply by aneisch** [reply]
Hmm, I have an Amcrest ASH21 and can't seem to get it going.

**Reply by NickM-27** [reply]
that camera does not support onvif https://amcrest.com/downloadable/download/attachment/id/41929/

**Reply by KenwoodFox** [reply]
Glad i saw this, i was just thinking of buying an ASH21.

**Reply by hawkeye217** [reply]
Amcrest and Dahua have a consumer line of cameras that have a smaller feature set than their higher end models. From what other users have reported, the ASH41 and ASH47 can be manually controlled but lack the features necessary for autotracking in frigate.

**Comment by Cold-Lemonade** [comment]
I have a Dahua PTZ1A2M-25X which I bought from EmpireTech (on [Amazon](https://www.amazon.com/dp/B08LGVS1RR)). I'm running Frigate v0.12.1 and it works fine. I have control it through buttons that I defined in Home Assistant that issue commands to the camera.

**Comment by jpettitt** [comment]
The Reolink 823a will not do tracking in Frigate because it can't for FOV relative pan/tilt.

**Comment by maxi1134** [comment]
Anyone knows of a cheap ( sub 500$) that can do tracking with Frigate?
I cannot find a specific model and don't wanna be stuck with a camera.

I currently have a reolink 823 that does not offer relative pan/tilt and I am looking to replace it!

**Reply by hawkeye217** [reply]
I developed autotracking with the Dahua SD1A404XB-GNR, it's a sub-$200 camera.

**Reply by maxi1134** [reply]
Sorry, I should've mentioned that I am solely looking for POE. This one seems to be a wifi camera

**Reply by hawkeye217** [reply]
No, it's PoE

**Reply by maxi1134** [reply]
Oh I see, it gave me the -w version on google.

This is for outside. I will check if it is rated for outdoors and order one if I can find one on a reputable seller. So far I only find them on Aliexpress

(I don't trust aliexpress for anything over 20$)

**Reply by hawkeye217** [reply]
The SD1A404XB-GNR is an outdoor camera.

**Reply by hawkeye217** [reply]
Would recommend Andy at EmpireTech: https://empiretech01.com/
Looks like they have a refurbished model on sale right now.

**Reply by maxi1134** [reply]
> Would recommend Andy at EmpireTech: https://empiretech01.com/ Looks like they have a refurbished model on sale right now.

I am ready to order, after 2 person tried to break into my house several times this week end.

Would this be compatible with the PTZ and tracking?

https://empiretech01.com/products/empiretech-ptz3e405-ai-1-2-8cmos-4mp-5x-tioc-ptz-camera

**Reply by hawkeye217** [reply]
None of the maintainers have that camera, so we can't tell you for sure. Maybe a user has it?

**Reply by maxi1134** [reply]
Unfortunately, no one seems to do. I might order it and let you guys know!

I sent an email to Andy, but he was unable to test the cameras with the script for me.
